### PR TITLE
feat: add MetaStreetConfig

### PR DESCRIPTION
**Pull request by aouahib**

### CODE REVIEW

**Comment by aouahib** [comment]
Current dependencies on/for this PR:
* main
  * **PR #170** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/170" target="_blank">Graphite</a>  👈
    * **PR #171** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/171" target="_blank">Graphite</a> 
      * **PR #172** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/172" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/170?utm_source=stack-comment).

**Review by andrewszucs — APPROVED** [review]

**Comment by aouahib** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/170?utm_source=stack-comment).


Job Reference: Il3EKAhxbQsIJwI9q34e

**Pull request merged**
